mbot:
  # Publish all joint states -----------------------------------
  mbot_joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  z_rotator_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: z_rotator_joint
    pid: {p: 10.0, i: 0.001, d: 0.0,i_clamp_min: 0.0, i_clamp_max: 1.0}
  y_rotator_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: y_rotator_joint
    pid: {p: 0.05, i: 0, d: 0.01,i_clamp_min: 0.0, i_clamp_max: 10.0}
    #pid: {p: 9.6, i: 2.0, d: 0.0}

  left_wheel_joint_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: left_wheel_joint
    pid: {p: 0.3, i: 0.0, d: 0.0,i_clamp_min: 0.0, i_clamp_max: 1.0}
  right_wheel_joint_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: right_wheel_joint
    pid: {p: 0.3, i: 0.0, d: 0.0,i_clamp_min: 0.0, i_clamp_max: 1.0}


